plugins/europeana/record.js

```diff
@@ -37,7 +37,11 @@ function parseRecordDataFromApiResponse(response) {
     media: providerAggregation.webResources.filter((webResource) => {
       return (webResource.about === providerAggregation.edmIsShownBy) ||
         (providerAggregation.hasView || []).includes(webResource.about);
-    })
+    }),
+    title: providerProxy.dcTitle,
+    edmIsShownBy: providerAggregation.webResources.find((webResource) => {
```

Comment:
This needs to be re-removed. edm:isShownBy is included in the media array as the first item.